app/controllers/course/experience_points_records_controller.rb

```diff
@@ -8,4 +8,16 @@ def index # :nodoc:
     @experience_points_records =
       @experience_points_records.active.includes(creator: :course_users)
   end
+
+  def destroy # :nodoc:
```

Comment:
break up the destroy action into private destroy_success and destroy_failure methods.